src/Network/GRPC/HighLevel/Generated.hs

```diff
@@ -28,10 +28,17 @@ module Network.GRPC.HighLevel.Generated (
 
   -- * Server Auth
 , ServerSSLConfig(..)
+
+  -- * Client
```

Comment:
I just looked at the cabal file and realized its version is zeroed out, do you know why we did that? @Gabriel439 is the intention behind that because we haven't made an official release of it yet?